### PR TITLE
[DUOS-1758][risk=no] Data Use Agreements - Basic Panel UI

### PR DESCRIPTION
**Pull request by connorlbark**
### Addresses

https://broadworkbench.atlassian.net/browse/DUOS-1758

----
Have you read [Terra's Contributing Guide](https://github.com/DataBiosphere/terra-ui/wiki/Contributor-Guide) lately? If not, do that first.

- Label PR with a Jira ticket number and include a link to the ticket
- Label PR with a security risk modifier [no, low, medium, high]
- PR describes scope of changes
- Get a minimum of one thumbs worth of review, preferably two if enough team members are available
- Get PO sign-off for all non-trivial UI or workflow changes
- Verify all tests go green
- Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

**Review by rushtong — APPROVED** [review]
Looks good 👍🏽

File: src/pages/DataAccessRequestApplication_new.js [hunk 1]
```diff
@@ -959,18 +959,8 @@ class DataAccessRequestApplicationNew extends Component {
...
-                problemSavingRequest,
-                attestAndSave: this.attestAndSave,
-                ConfirmationDialogComponent,
-                partialSave: this.partialSave,
-                prevPage: this.prevPage,
-                step1Invalid,
-                step2Invalid,
-                step3Invalid,
-                libraryCardInvalid,
-                showValidationMessages,
```
Comment (quazi-h) [inline src/pages/DataAccessRequestApplication_new.js:971]: Are we removing the invalid checks and problemSavingRequest and ConfirmationDialogComponent completely? When running locally my console shows a few warning about these being assigned but never used.

File: src/pages/dar_application/DataUseAgreements_new.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    attestAndSend,
+    darCode,
+  } = props;
+
+  return (
+    div({
+      className: 'dar-step-card'
+    }, [
+
+      h2({}, ['Step 4: Attestation and Agreement(s)']),
```
Comment (quazi-h) [inline src/pages/dar_application/DataUseAgreements_new.js]: This is slightly different from the mockup which has "Data Use Agreements" for the header

**Review by quazi-h — APPROVED** [review]
Changes look good to me. I left comments about some unused variables and a just double checking that the header for the section is correct but not a blocker for this PR.

File: src/pages/DataAccessRequestApplication_new.js (hunk 1 above)
Comment (connorlbark) [reply src/pages/DataAccessRequestApplication_new.js:971]: Good point, I can remove those. This is part one of the PR, so I didn't want to do too much, but I don't want to add linting issues to develop, so I will get rid of those.

**Comment by connorlbark** [comment]
hey @quazi-broad I fixed your feedback + made a styling change, could you give a quick re-review?

**Review by quazi-h — APPROVED** [review]
Thanks for the changes, looks good!

**Pull request merged**